src/api/validations/auth.validation.js

```diff
@@ -33,12 +33,35 @@ module.exports = {
         .email()
         .required(),
       url: Joi.string()
-        .uri()
+        // .uri()
         .trim()
         .required(),
     },
   },
 
+  passwordResetChange: {
```

Comment:
Looks good that we have password now, this means we'll have to update this on the client end too, correct?

Reply:
Yes, so moving the client apps to support this basic api-boilerplate is definitely something to move toward (at least for projects that would use this boilerplate).